### PR TITLE
games-strategy/liquidwar: version bump

### PR DESCRIPTION
**Pull request by DarthGandalf**
Closes: https://bugs.gentoo.org/707810
Package-Manager: Portage-2.3.103, Repoman-2.3.23
Signed-off-by: Alexey Sokolov <alexey+gentoo@asokolov.org>

### CODE REVIEW

File: games-strategy/liquidwar/liquidwar-5.6.5.ebuild [hunk 1]
```diff
@@ -1,51 +1,53 @@
...
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit desktop
+EAPI=7
+
+inherit desktop toolchain-funcs
 
 DESCRIPTION="Unique multiplayer wargame"
 HOMEPAGE="http://www.ufoot.org/liquidwar/"
```
Comment (steils) [inline games-strategy/liquidwar/liquidwar-5.6.5.ebuild:9]: You can use https here

File: games-strategy/liquidwar/liquidwar-5.6.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+src_compile() {
+	# skip build_doc target wrt bug 460344
+	emake build_bin build_data
+}
+
+src_install() {
+	emake DESTDIR="${D}" install_nolink
+	einstalldocs
+	rm -f "${ED}"/usr/share/doc/${PF}/COPYING
```
Comment (steils) [inline games-strategy/liquidwar/liquidwar-5.6.5.ebuild]: `|| die`, please.
and no need for `-f`.
And if possible to remove the file in src_prepare, please do so.

File: games-strategy/liquidwar/liquidwar-5.6.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+	# Provided desktop file is completely obsolete
+	rm -f "${ED}"/usr/share/applications/liquidwar.desktop
```
Comment (steils) [inline games-strategy/liquidwar/liquidwar-5.6.5.ebuild]: same here

**Review by steils — CHANGES_REQUESTED** [review]

File: games-strategy/liquidwar/liquidwar-5.6.5.ebuild (hunk 2 above)
Comment (DarthGandalf) [reply games-strategy/liquidwar/liquidwar-5.6.5.ebuild]: Done. Also simplified src_prepare

**Review by steils — APPROVED** [review]

**Pull request closed without merging**